etc/kayobe/kolla/inventory/overcloud-components.j2

```diff
@@ -0,0 +1,280 @@
+# This inventory section provides a mapping of the components to top level
```

Comment:
Sure we want this? alt-1 monitoring is on the controller, prod monitoring is on a separate host.

Reply:
we define:
```
kolla_overcloud_inventory_top_level_group_map:
  control:
    groups:
      - controllers
  network:
    groups:
      - network
  compute:
    groups:
      - compute
  monitoring:
    groups:
      - controllers
```
https://github.com/SKA-ScienceDataProcessor/alaska-kayobe-config/blob/29429c5264eb0d6eed20b668231bc7ac357d23b3/etc/kayobe/kolla.yml#L214

So it should be equivalent, right? My opinion is that the more similar the config is the better.